server_admin/topics/events/login.adoc

```diff
@@ -142,8 +190,7 @@ that comes with your distribution and adding for example:
 </spi>
 ----
 
-You can also set up a maximum length of the Event detail stored in the database by editing `standalone.xml`, `standalone-ha.xml`, or 
-`domain.xml`. This setting can be useful in case some field (e.g. redirect_uri) is very long. Here is an example of defining the maximum length.:
+You can set a maximum length of time the event detail stores in the database by editing the `standalone.xml`, `standalone-ha.xml`, or `domain.xml` configuration files. This setting is useful if a field (for example, redirect_uri) is long. For example:
```

Comment:
I think it's the maximum field length, not maximum time.
You can set a maximum length of the Event detail that is stored in the database